### ISSUE TITLE
Adding a new "Components" tag 

### ISSUE THREAD

**Issue by ZahidMKhan** [baynezy/Xunit.OpenCategories#63]
Hi,
I would like to propose a new tag "Components", very similar to Component but allows multiple value within one key.
I am kind of done with the changes and ready to push, but I am unfortunately unable to both fork this repo and create a pull request(_this one of kind of makes sense_).

Happy to come onboard as a maintainer if you'd like. I have more ideas for this.

@baynezy

**Comment by baynezy** [comment]
@ZahidMKhan yeah you can only fork a fork if you haven't already forked the parent (don't read this sentence out loud as it sounds mental). 

So we have 2 options. 

1. You delete your fork of the parent and fork mine. 
2. You send me the link to your branch and I'll pull it over.

**Comment by ZahidMKhan** [comment]
Thanks. I will fork replace the fork and open a PR.

**Comment by ZahidMKhan** [comment]
updated here #65
